isy-serviceapi-core/src/test/java/de/bund/bva/isyfact/serviceapi/core/httpinvoker/TimeoutWiederholungHttpInvokerRequestExecutorTest.java

```diff
@@ -20,33 +25,42 @@
  */
 public class TimeoutWiederholungHttpInvokerRequestExecutorTest {
 
-    /** Some value for the content length, doesn't get actually used. */
+    /**
+     * Some value for the content length, doesn't get actually used.
+     */
     private static final int CONTENT_LENGTH = 1;
 
-    /** AufrufKontextVerwalter to set bearer token in. */
+    /**
+     * AufrufKontextVerwalter to set bearer token in.
+     */
     private AufrufKontextVerwalter<?> aufrufKontextVerwalterStub;
 
-    /** Stubbed request executor. */
+    /**
+     * Stubbed request executor.
+     */
     private TimeoutWiederholungHttpInvokerRequestExecutor executorStub;
 
-    /** Stubbed connection that doesn't connect to anything. */
+    /**
+     * Stubbed connection that doesn't connect to anything.
+     */
     private HttpURLConnection connection;
 
     @Before
     public void setUp() {
         aufrufKontextVerwalterStub = new AufrufKontextVerwalterStub<>();
         aufrufKontextVerwalterStub.setBearerToken(null);
-        executorStub = new TimeoutWiederholungHttpInvokerRequestExecutorStub(aufrufKontextVerwalterStub);
+        SecurityContextHolder.createEmptyContext();
+        executorStub = new TimeoutWiederholungHttpInvokerRequestExecutorStub(null);
```

Comment:
`AufrufKontextVerwalter` and `SecurityContextHolder` are set for all testcases.
So the changed logic to set the baererToken from the `SecurityContextHolder` in the case no `AufrufKontextVerwalter` exists is not tested.

Reply:
The TestCase was updated by moving the initialization of executorStub and aufrufKontextVerwalterStub from the setUp
method to specific individual test methods.

Reply:
solved...